src/views/AnnualReport.vue

```diff
@@ -569,71 +588,56 @@ export default {
               }
             }
 
-            // load Annual Report fields
-            const annualReport = filing.annualReport
-            if (annualReport) {
-              // set the Draft Date in the Directors List component
+            if (header.effectiveDate) {
+              const effectiveDate = this.convertUTCTimeToLocalTime(header.effectiveDate).slice(0, 10)
+              // restore Draft Date in Directors List component
               // FUTURE: use props instead of $refs (which cause an error in the unit tests)
-              if (this.$refs.directorsList && this.$refs.directorsList.setDraftDate) {
-                this.$refs.directorsList.setDraftDate(annualReport.annualGeneralMeetingDate)
+              if (this.$refs.directorsList?.setDraftDate) {
+                this.$refs.directorsList.setDraftDate(effectiveDate)
               }
-              if (this.isCoop) {
-                // set the new AGM date in the AGM Date component (may be null or empty)
-                this.newAgmDate = annualReport.annualGeneralMeetingDate || ''
-                // set the new No AGM flag in the AGM Date component (may be undefined)
-                this.newNoAgm = annualReport.didNotHoldAgm || false
+            } else if (annualReport.annualGeneralMeetingDate) {
+              // restore Draft Date in Directors List component
+              // FUTURE: use props instead of $refs (which cause an error in the unit tests)
+              if (this.$refs.directorsList?.setDraftDate) {
+                this.$refs.directorsList.setDraftDate(annualReport.annualGeneralMeetingDate)
               }
             } else {
-              throw new Error('Missing annual report')
+              // eslint-disable-next-line no-console
+              console.log('fetchData() error = missing Effective Date')
+            }
+
+            // load Annual Report fields
+            if (this.isCoop) {
+              // set the new AGM date in the AGM Date component (may be null or empty)
+              this.newAgmDate = annualReport.annualGeneralMeetingDate || ''
+              // set the new No AGM flag in the AGM Date component (may be undefined)
+              this.newNoAgm = annualReport.didNotHoldAgm || false
             }
 
             // load Change of Directors fields
             const changeOfDirectors = filing.changeOfDirectors
             if (changeOfDirectors) {
-              if (changeOfDirectors.directors && changeOfDirectors.directors.length > 0) {
-                if (this.$refs.directorsList && this.$refs.directorsList.setAllDirectors) {
+              if (changeOfDirectors?.directors.length > 0) {
+                if (this.$refs.directorsList?.setAllDirectors) {
                   this.$refs.directorsList.setAllDirectors(changeOfDirectors.directors)
                 }
-
-                // add filing code for paid changes
-                if (changeOfDirectors.directors.filter(
```

Comment:
This is unnecessary code and caused duplicate filing data updates.
The Directors component emits events back to THIS file to set the filing data accordingly.